13/umbraco-forms/developer/working-with-data.md

```diff
@@ -77,7 +77,7 @@ Guid UniqueId
 Dictionary<Guid, RecordField> RecordFields
 ```
 
-In order to access custom Form fields, these are available in the `RecordFields` property. Furthermore there exists an extension method named `ValueAsString` on `IRecord` in `Umbraco.Forms.Core.Services`, such that you can get the value as string given the alias of the field.
+In order to access custom Form fields, these are available in the `RecordFields` property. Furthermore there exists an extension method named `ValueAsString` on `Record` in `Umbraco.Forms.Core.Services`, such that you can get the value as string given the alias of the field.
 
 This extension method handle multi value fields by comma separating the values. E.g. "A, B, C"
 
```

Comment:
Should `@using Umbraco.Forms.Core.Services;` in sample code be changed to `@using Umbraco.Forms.Core.Extensions;` as done in v14 article?

Reply:
No, not for 13.  This moved namespace in 14 so I deliberately only updated in there.

The change on this page is just `IRecord` --> `Record`, which should really have been done when Forms 13 came out, but it wasn't noticed on this page.

Reply:
Ah ok, Got it 💪